series-tvdb.en.yaml

```diff
@@ -1054,6 +1078,14 @@ entries:
         anilist-id: 20993
         start: 13
 
+  - title: "Kawaii dake ja Nai Shikimori-san"
```

Comment:
The English title should be in `title:`, the Japanese one in synonyms.

Reply:
Aye, no problem.